deploy/kubernetes/clusterrole.yaml

```diff
@@ -107,6 +107,7 @@ rules:
 - apiGroups: ["rbac.istio.io"]
   resources:
   - clusterrbacconfigs
+  - rbacconfigs
```

Comment:
@xeviknal can you create a JIRA in Maistra to include this in their distribution ?
Also @jmazzitelli and @gbaufake need to be updated that upstream will require changes once this version is deployed on istio.

Reply:
https://issues.jboss.org/browse/KIALI-2564